package.json

```diff
@@ -1,4 +1,16 @@
 {
+  "name": "Short",
+  "version": "0.0.4",
+  "description": "Visit short links with less typing",
+  "repository": {
+    "type": "git",
+    "url": "https://github.com/byliuyang/short-ext.git"
+  },
+  "scripts": {
+    "dev": "yarn run cp && tsc --watch",
+    "build": "yarn run cp && tsc",
+    "cp": "cp -r icons extension/ && cp manifest.json extension/"
```

Comment:
```suggestion
    "cp:assets": "cp -r icons  manifest.json build/"
```

Reply:
cant run `cp -r icons manifest.json build/`

Reply:
@fxnoob It works for me on macOS. Did you create ```build/``` directory first?

Reply:
Please try this: 
```json
"scripts": {
    "build": "yarn prep && yarn cp:assets && tsc",
    "prep": "rm -rf build && mkdir build",
    "dev": "yarn build --watch",
    "cp:assets": "cp -r icons manifest.json build/"
  },
```